### PR TITLE
added swacrew files

### PR DESCRIPTION
**Pull request by SWACOHD**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/swacrew/manifest.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  "server_wildcards": [
+    "%.swacrew.net",
+    "%.swacrew.de"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://swacrew.net",
```
Comment (DoJapHD) [inline minecraft_servers/swacrew/manifest.json:14]: SSL handshake failed

File: minecraft_servers/swacrew/manifest.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    "twitter": "SwacrewNET",
+    "tiktok": "swacrew",
+    "youtube": "https://www.youtube.com/swacrewnet",
+    "discord": "https://discord.gg/JgFFcVRKed",
+    "teamspeak": "Swacrew.net"
+  },
+  "gamemodes": {
+    "eventserver": {
+      "name": "Eventserver",
+      "url": "https://www.swacrew.net",
```
Comment (DoJapHD) [inline minecraft_servers/swacrew/manifest.json:25]: Remove URL parameter in gamemodes (all!)

File: minecraft_servers/swacrew/manifest.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    "vorbau": {
+      "name": "Vorbau",
+      "url": "https://www.swacrew.net",
+      "command": "/gameteleport vorbau"
+    }
+  },
+  "discord": {
+    "server_id": 746838764914999431,
+    "rename_to_minecraft_name": true
```
Comment (DoJapHD) [inline minecraft_servers/swacrew/manifest.json:66]: Only for server partner, set to false

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/swacrew/manifest.json (hunk 1 above)
Comment (SWACOHD) [reply minecraft_servers/swacrew/manifest.json]: removed

File: minecraft_servers/swacrew/manifest.json (hunk 2 above)
Comment (SWACOHD) [reply minecraft_servers/swacrew/manifest.json]: done

File: minecraft_servers/swacrew/manifest.json (hunk 3 above)
Comment (SWACOHD) [reply minecraft_servers/swacrew/manifest.json]: done

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**